tests/stress/stress_test.go

```diff
@@ -641,3 +642,64 @@ func TestSync_SubmitExtrinsicLoad(t *testing.T) {
 		// Output: Balance transferred from Alice to Bob: 100000000000000
 	}
 }
+
+func TestSecondarySlotProducer(t *testing.T) {
+	numNodes := 4
+	utils.Logger.Patch(log.SetLevel(log.Info))
+
+	// start block producing node first
+	node, err := utils.RunGossamer(t, numNodes-1,
+		utils.TestDir(t, utils.KeyList[numNodes-1]),
+		utils.GenesisDev, utils.ConfigNoGrandpa,
+		false, true)
+	require.NoError(t, err)
+
+	// wait and start rest of nodes - if they all start at the same time the first round usually doesn't complete since
+	// all nodes vote for different blocks.
+	time.Sleep(time.Second * 15)
+	nodes, err := utils.InitializeAndStartNodes(t, numNodes-1, utils.GenesisDev, utils.ConfigNotAuthority)
+	require.NoError(t, err)
+	nodes = append(nodes, node)
+
+	time.Sleep(time.Second * 30)
+
+	defer func() {
+		errList := utils.StopNodes(t, nodes)
+		require.Len(t, errList, 0)
+	}()
+
+	numCmps := 10
+	secondaryCount := 0
+
+	for i := 0; i < numCmps; i++ {
+		time.Sleep(3 * time.Second)
```

Comment:
Do we need these 3s sleeps?

Reply:
test logic was inappropriate, I have changed it